recipes/cylc-flow/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "cylc-flow" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "36f4f4363cb5cc7a3a613d2db6e84cc41c2825c6cd011191cc4821c5ca9c81a4"
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - ansimarkup >=1.0.0
+    - colorama >=0.4
+    - graphene ==2.1.6
+    - jinja2 >=2.10.1
+    - markupsafe >=1.1
+    - metomi-isodatetime >=1!2.0.1
+    - pip
+    - protobuf >=3.7
+    - pytest-runner
```

Comment:
We should be able to get away without installing `pytest` for normal builds, perhaps remove or put in a `test` section?

Reply:
Good spot & thinking, I will adjust that now, & see if the checks still pass.

Reply:
Oh, I think it was also fixed in metomi-isodatetime by making it an optional dependency there: https://github.com/metomi/isodatetime/pull/143

Next release of metomi-isodatetime conda recipe can have pytest-runner and license removed. Just pending a release to PYPI first.